ios/TransactionListViewCell.swift

```diff
@@ -43,6 +43,9 @@ class TransactionListViewCell: TransactionListBaseCell {
     if transaction.symbol != nil {
       if let img = UIImage.init(named: transaction.symbol.lowercased()) {
         coinImage.image = img
+      } else if transaction.address != nil {
+        let url = URL(string: "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/\(transaction.address!)/logo.png");
+        coinImage.sd_setImage(with: url)
```

Comment:
There's one more thing left. On the JS side, we have a listener for the image onError event, and in that case we fallback to the case that you have below: See https://github.com/rainbow-me/rainbow/blob/develop/src/components/coin-icon/CoinIcon.js#L71

Reply:
Oh, I see! You're right, we need to handle the fetch error for sure. Will fix it ASAP